internal/translation/dbuser/conversion_test.go

```diff
@@ -7,6 +7,8 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	corev1 "k8s.io/api/core/v1"
```

Comment:
I think the issue here is `goimports` notices when you put the import in a group it does not belong to, but rather than fix it on its own, is takes the line out so that you would put it in the right place.

The right place is not as simple as "move it to line 9", the right place is move it to the right block of imports and let `goimports` sort it in there.

We seems to be using 3 blocks of imports, that `goimports` will respect:
1. Go stdlb imports
2. Our code
3. Other libraries (including other libraries by MongoDB, basically any other repo)

Unfortunately, we do not follow the ordering consistently, sometimes it is:
1. Go
2. Ours
3. Theirs

Others is Ours and Theirs are swapped 

I suggest we stick with the above sample order, 3 blocks Go/Ours/Theirs